src/widget/wcoverart.cpp

```diff
@@ -265,7 +265,7 @@ void WCoverArt::mouseReleaseEvent(QMouseEvent* event) {
             m_clickTimer.isActive()) { // init/close fullsize cover
         if (m_pDlgFullSize->isVisible()) {
             m_pDlgFullSize->close();
-        } else {
+        } else if (!m_loadedCover.isNull()) {
```

Comment:
This conditional is confusing because the member is apparently unrelated to the following function call. Do we really need this check?

I would prefer to handle this special case solely in DlgCoverArtFullSize if possible, even when this requires a function call that immediately returns.

Reply:
I'm also not super happy about this, but `init` already shows the window and then emits a signal to load the cover. This was an "easy" fix to not even show a fullsize window if the track has no cover.

Reply:
Please add this as a comment.

Reply:
https://github.com/mixxxdj/mixxx/pull/3070/commits/82aac90ebe7842fd11fb2086884bb3e2939fdcab